README.md

```diff
@@ -26,10 +26,7 @@
 - `pipenv install` (in the repo's ROOT folder)
 - `cp similarity/local_config.py.example local_config.py` (then set the correct configuration of your local machine).
 - `cp _example.env .env`
-- `pipenv shell` (optional for the next commands; instead of adding `pipenv run` in the beginning for each one):
-   - `flask db init` (initialize local db)
-   - `flask db migrate -m "Initial db migration"`
-   - `flask db upgrade`
+- `bash ./initiate_local_db.sh`
```

Comment:
@ordavidil should it be ran with/without pipenv (i.e. pipenv shell / pipenv run ....)?